trie/verkle.go

```diff
@@ -163,14 +163,14 @@ func (t *VerkleTrie) UpdateAccount(addr common.Address, acc *types.StateAccount,
 		stem      = t.pointCache.GetTreeKeyBasicDataCached(addr[:])
 	)
 
-	binary.LittleEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
+	binary.BigEndian.PutUint64(basicData[utils.BasicDataNonceOffset:], acc.Nonce)
 	// get the lower 16 bytes of water and change its endianness
 	balanceBytes := acc.Balance.Bytes()
 	for i := 0; i < 16 && i < len(balanceBytes); i++ {
-		basicData[utils.BasicDataBalanceOffset+i] = balanceBytes[len(balanceBytes)-1-i]
+		basicData[utils.BasicDataBalanceOffset+i] = balanceBytes[i]
 	}
 	// var cs [8]byte
-	// binary.LittleEndian.PutUint64(cs[:], uint64(codeLen))
+	// binary.BigEndian.PutUint64(cs[:], uint64(codeLen))
```

Comment:
yeah we don't even write the code size, my code is definitely broken. I have a tentative fix in my replay machine, I hope to be able to validate it today.